lighthouse-core/audits/largest-contentful-paint-element.js

```diff
@@ -60,7 +60,7 @@ class LargestContentfulPaintElement extends Audit {
 
     const details = Audit.makeTableDetails(headings, lcpElementDetails);
 
-    const displayValue = str_(i18n.UIStrings.displayValueElementsFound, {itemCount: lcpElementDetails.length});
+    const displayValue = str_(i18n.UIStrings.displayValueElementsFound, {nodeCount: lcpElementDetails.length});
```

Comment:
this was due to the change i made you make in this file.  the message string expected something slightly diff.

noticed cuz of this error: 
![image](https://user-images.githubusercontent.com/39191/90833564-28f57680-e2fd-11ea-80cd-514ea0a65af0.png)